lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -5,129 +5,291 @@
+    const minimalProps: Props = {
+          value: 0
+        },
+        getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxBinaryDonutChart, minimalProps);
 
-  it('renders an svg with the expected properties', function() {
-    expect(getShallowComponent({ percent: 90 })).toHaveClassName('.nx-binary-donut-chart');
-    expect(getShallowComponent({ percent: 90 })).toHaveProp('viewBox', '-15 -15 30 30');
-    expect(getShallowComponent({ percent: 90 })).toMatchSelector('svg');
-  });
+    it('sets the role to "meter"', function() {
+      expect(getShallowComponent()).toHaveProp('role', 'meter');
+    });
 
-  it('renders only the background path with the expected properties at zero percent', function() {
-    const circles = getShallowComponent().find('circle');
-    const paths = getShallowComponent().find('path');
+    it('sets the aria-valuemin to 0', function() {
+      expect(getShallowComponent()).toHaveProp('aria-valuemin', 0);
+    });
 
-    expect(circles.length).toBe(0);
-    expect(paths.length).toBe(1);
-    expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    it('sets the aria-valuemax to 100 when maxVal is not specified', function() {
+      expect(getShallowComponent()).toHaveProp('aria-valuemax', 100);
+    });
 
-    // whether the background renders correctly is best left to the visual tests
-  });
+    it('sets the aria-valuemax to maxVal when maxVal is specified', function() {
+      expect(getShallowComponent({maxVal: 200})).toHaveProp('aria-valuemax', 200);
+    });
 
-  it('renders the background path and an arc circle with the expected properties when percent is 100', function() {
-    const circles = getShallowComponent({ percent: 100}).find('circle');
-    const paths = getShallowComponent({ percent: 100}).find('path');
+    it('sets the aria-valuenow to value', function() {
+      expect(getShallowComponent({value: 30})).toHaveProp('aria-valuenow', 30);
+    });
 
-    expect(circles.length).toBe(1);
-    expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__arc');
-    expect(circles.at(0)).toHaveProp('strokeWidth', 7);
-    expect(circles.at(0)).toHaveProp('r', 10.5);
-    expect(paths.length).toBe(1);
-    expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
-  });
+    it('renders an svg with the expected properties', function() {
+      expect(getShallowComponent({value: 90})).toHaveClassName('.nx-binary-donut-chart');
+      expect(getShallowComponent({value: 90})).toHaveProp('viewBox', '-15 -15 30 30');
+      expect(getShallowComponent({value: 90})).toMatchSelector('svg');
+    });
 
-  it('renders only a background circle path percent is negative', function() {
-    const circles = getShallowComponent({ percent: -50}).find('circle');
-    const paths = getShallowComponent({ percent: -50}).find('path');
+    it('renders only the background path with the expected properties at zero percent', function() {
```

Comment:
A bunch of these test names are still talking about percent rather than value

Reply:
The test names here are under describe `'with non-deprecated props'` block, it tests at certain `percent` calculated based on value and default/non-default maxVal. But it might not be clear, I will change it to `'renders only the background path with the expected properties at zero value'`.

Reply:
This is updated here f82507e6b150c4c93b8fcfba96d44f31ca682ddb